src/Plugin/Visualisation/Style/AxisChart.php

```diff
@@ -662,6 +690,38 @@ public function removeGridLine(array &$form, FormStateInterface $form_state) {
     $form_state->setRebuild();
   }
 
+  /**
+   * Ajax callback that updates the column overrides options.
+   *
+   * @param array $form
+   *   The form structure.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current state of the form.
+   *
+   * @return array
+   *   The updated form element.
+   */
+  public function updateColumnOverrides(array $form, FormStateInterface $form_state) {
+    $form_state->setRebuild(TRUE);
```

Comment:
Just curious but why do you need to trigger a rebuild?

Reply:
For whatever reason I thought these was required before returning the updated form element. Anyway, it works fine without it so I have removed the two instances of `rebuild()`